charts/kubeai/values.yaml

```diff
@@ -60,11 +60,26 @@ resourceProfiles:
   #   nodeSelector:
   #     cloud.google.com/gke-accelerator-type: tpu-v5-lite-podslice
 
+autoscaling:
+  interval: 10s
+  timeWindow: 10m
```

Comment:
Can you add a comment about how interval and timeWindow work?

Reply:
I often do `helm show values` to see what values are available and having doc strings helps a lot.

Reply:
done